packages/nouns-api/src/clients.ts

```diff
@@ -1,17 +1,20 @@
 import { config } from './config';
-import { Contract, providers } from 'ethers';
+import { Contract, providers, getDefaultProvider } from 'ethers';
 import { NounsTokenABI } from '@nouns/contracts';
 
 /**
  * Ethers JSON RPC Provider
  */
 export const jsonRpcProvider = new providers.JsonRpcProvider(config.jsonRpcUrl);
+export const defaultProvider = getDefaultProvider(1, {
```

Comment:
@ThatTobMate I'm not super savvy with the ethereum stuff, but the JSON rpc was not working correctly, and we needed some sort of provider in order to query the chain to answer questions like "how many lil nouns does this address hold"

If we can figure out the jsonRPC then this would work fine, otherwise we may have to use something like infura, which is what I've done here, although I'm not sure how that scales in a production env with many requests (might have to pay?)